istioctl/cmd/revision.go

```diff
@@ -118,14 +118,14 @@ func revisionDescribeCommand() *cobra.Command {
 	describeCmd := &cobra.Command{
 		Use: "describe",
 		Example: `  # View the details of a revision named 'canary'
-  istioctl experimental revision describe canary
+  istioctl x revision describe canary
```

Comment:
Thanks for the PR! 
Do you only want to gradute `istioctl x revison tag` but keep `istioctl x revison describe/list` in experimental ?  Any reaosn for only graduate part of the commands?   Seems the release notes does not match this either.

One other issue: `tag` command shows up duplicate in `help` output. 
```
istioctl --help
Istio configuration command line utility for service operators to
debug and diagnose their Istio mesh.

Usage:
  istioctl [command]

Available Commands:
  admin          Manage control plane (istiod) configuration
  analyze        Analyze Istio configuration and print validation messages
  authz          (authz is experimental. Use `istioctl experimental authz`)
  bug-report     Cluster information and log capture support tool.
  dashboard      Access to Istio web UIs
  experimental   Experimental commands that may be modified or deprecated
  help           Help about any command
  install        Applies an Istio manifest, installing or reconfiguring Istio on a cluster.
  kube-inject    Inject Istio sidecar into Kubernetes pod resources
  manifest       Commands related to Istio manifests
  operator       Commands related to Istio operator controller.
  profile        Commands related to Istio configuration profiles
  proxy-config   Retrieve information about proxy configuration from Envoy [kube only]
  proxy-status   Retrieves the synchronization status of each Envoy in the mesh [kube only]
  tag            Command group used to interact with revision tags
  tag            Command group used to interact with revision tags
  upgrade        Upgrade Istio control plane in-place
  validate       Validate Istio policy and rules files
  verify-install Verifies Istio Installation Status
  version        Prints out build version information
```

Reply:
Yes, the agreement from discussion in UX working group meeting was to move `tag` but leave `describe/list` since those commands are less stable.

Thanks for catching the duplicate, just fixed.

Reply:
Thanks for the fix!  Now we make `istioctl tag` command a seperate command group - Can you update the release notes to reflect this?  We should not say we ``istioctl experimental revision` command group to `istioctl revision`.`

Reply:
Just updated the release note, thanks!